mkdocs.yml

```diff
@@ -31,7 +31,9 @@ nav:
     - Usage: test-suite/usage.md
     - Release notes: test-suite/release-notes.md
   - Build-test-deploy bot: bot.md
-  - Pilot repository: pilot.md
```

Comment:
add a redirect, we don't want https://www.eessi.io/docs/pilot to be a broken URL, it should redirect to https://www.eessi.io/docs/repositories/pilot

Reply:
added and tested redirect